core/src/main/java/org/elasticsearch/action/support/tasks/BaseTasksRequest.java

```diff
@@ -95,6 +96,22 @@ public final Request nodesIds(String... nodesIds) {
     }
 
     /**
+     * Returns the id of the task that should be processed.
+     *
+     * By default tasks with any ids are returned.
+     */
+    public long taskId() {
```

Comment:
This is ActionRequest and this is how we handle parameters in all our ActionRequests. I always hated this convention and I would love to use getTaskId and setTaskId there, but it's a convention. If we are officially going away from this convention, I will gladly change all task-management request to have standard setters and getters. Just let me know.